src/api/mapping.ts

```diff
@@ -15,7 +20,37 @@ mappingRouter.post('/', (req, res) => {
   }
   domainKeys.push(mappingObject)
   setData('mappings', domainKeys)
-  res.json(mappingObject)
+  const fullDomain = `${req.body.subDomain}.${req.body.domain}`
+  const prodConfig = {
+    apps: [
+      {
+        name: `${fullDomain}`,
+        script: './build/app.js',
+        instances: 1,
+        autorestart: true,
+        watch: false,
+        max_memory_restart: '1G',
+        env_production: {
+          NODE_ENV: 'production',
+          PORT: parseInt(req.body.port, 10)
+        }
+      }
+    ]
+  }
+  const projectPath = '~/projects'
+  const postReceiveScriptPath = `${path.join(__dirname, '../../scripts')}`
```

Comment:
`path.join(....)` is already a string right? Then its pointless to add `${....}`